docs/integrations/sources/amazon-ads.inapp.md

```diff
@@ -0,0 +1,20 @@
+## Prerequisites
+
+- An [Amazon user](https://www.amazon.com) with access to an [Amazon Ads account](https://advertising.amazon.com)
+
+## Setup Guide
+
+1. Click `Authenticate your Amazon Ads account`. Log in and authorize access to the Amazon account.
+2. (Optional) Select **Region** to pull data from **North America (NA)**, **Europe (EU)**, **Far East (FE)**. See [Amazon Ads documentation](https://advertising.amazon.com/API/docs/en-us/info/api-overview#api-endpoints) for more details.
+3. (Optional) **Start Date** can be used to generate reports starting from the specified start date in the format YYYY-MM-DD. The date should not be more than 60 days in the past. If not specified, today's date is used. The date is treated in the timezone of the processed profile.
+4. (Optional) **Profile ID(s)** you want to fetch data for. A profile is an advertiser's account in a specific marketplace. See [Amazon Ads docs](https://advertising.amazon.com/API/docs/en-us/concepts/authorization/profiles) for more details.
```

Comment:
as a user i'd want to know the behavior if I don't specify this optional field. Does it pull everything or nothing?